src/Streamly/Internal/Data/Array/Foreign/Mut/Type.hs

```diff
@@ -634,6 +699,11 @@ arrayChunkBytes = 1024
 -- Snoc
 -------------------------------------------------------------------------------
 
+{-# INLINE touch #-}
+touch :: ArrayContents -> IO ()
+touch (ArrayContents contents) =
```

Comment:
Add this on the top of the file? Define before use?

Reply:
Moved.